### PR TITLE
hometask_lesson_13

### PR DESCRIPTION
**Pull request by Giuseppe7777**
hometask_lesson_13

### CODE REVIEW

File: lesson_12/button.code-search
Comment (RomanIlkiv) [inline lesson_12/button.code-search:1]: what is that?:)

File: lesson_12/contact_us.html [hunk 1]
```diff
@@ -47,7 +47,7 @@
     <!-- Intro -->
     <div class="intro">
         <h1 class="intro__heading">Your favorite language school</h1>
-        <button class="btn btn-intro">Order lesson</button>
+        <button type="submit" onclick="location.href='contact_us.html'" class="btn btn-intro">Order lesson</button>
```
Comment (RomanIlkiv) [inline lesson_12/contact_us.html:50]: why so complicated?
you can use just `<a>`

File: lesson_13/gallery.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    <div class="wrapper">
+
+        <!-- Header -->
+        <header class="header">
+            <div class="header__worktime">
+                <p>Mo - Fr: 8:00 - 17:00</p>
+                <p>other days on request</p>
+            </div>
+            <div class="header__logo">
+                <a href="#"><img src="img/logo.png" alt=""></a>
```
Comment (RomanIlkiv) [inline lesson_13/gallery.html:25]: could be link to index.html
and why alt attribute is empty?

File: lesson_13/gallery.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        
+        <!-- Main -->
+        <main class="main">
+            <h1>Gallery</h1>
+            <div class="main__container">
+                <img src="img/school-images/school-01.jpeg" alt="">
+                <img src="img/school-images/school-02.jpeg" alt="">
+                <img src="img/school-images/school-03.jpg" alt="">
+                <img src="img/school-images/school-04.jpg" alt="">
+                <img src="img/school-images/school-05.jpg" alt="">
```
Comment (RomanIlkiv) [inline lesson_13/gallery.html:60]: add alt

Comment (RomanIlkiv) [reply lesson_13/gallery.html:60]: here and in other places

File: lesson_12/contact_us.html (hunk 1 above)
Comment (Giuseppe7777) [reply lesson_12/contact_us.html:50]: I've read about the difference between <button> and <a>. And then, I've chosen <button>. Is this wrong?

File: lesson_13/gallery.html (hunk 3 above)
Comment (Giuseppe7777) [reply lesson_13/gallery.html:60]: Ok!

File: lesson_13/gallery.html (hunk 2 above)
Comment (Giuseppe7777) [reply lesson_13/gallery.html:25]: Ok!

**Pull request merged**